README.md

```diff
@@ -32,10 +31,6 @@ A small app to explore Typescript, Vite and React.
 1. At the `sbl-frontend` command line, run `yarn start`. This script uses `docker-compose` to start up Docker containers of all of the project components (User management, API, Frontend) to get you up and running.
 1. At the `sbl-frontend` command line, run `yarn seed` to generate the necessary mock data in the backend systems.
 
-&NewLine;
-
-- This project uses yarn v3.5 in "plug n play" mode. There is no `node_modules/` directory. Packages are stored in `.yarn/cache/`.
-
```

Comment:
We[ have a ticket](https://github.com/cfpb/sbl-frontend/issues/831) to fully remove the weird cache stuff after we get our playwright tests all set.